src/main/java/ui/components/NavigableListView.java

```diff
@@ -158,11 +158,11 @@ protected void setupKeyEvents() {
             }
             if (FIRST_ISSUE.match(e)) {
                 e.consume();
-                selectFirstItem();
+                selectnthItem(1);
             }
             if (LAST_ISSUE.match(e)) {
                 e.consume();
-                selectLastItem();
+                selectnthItem(getItems().size() - 1);
```

Comment:
This doesn't look right, given that the method uses 1-based indexing.

Is there a test for this hotkey?